func.go

```diff
@@ -292,6 +317,351 @@ func RegisterFunc(fptr interface{}, cfn uintptr) {
 	fn.Set(v)
 }
 
+func addStruct(v reflect.Value, numInts, numFloats, numStack *int, addInt, addFloat, addStack func(uintptr), keepAlive []interface{}) []interface{} {
+	if v.Type().Size() == 0 {
+		return keepAlive
+	}
+	if runtime.GOARCH == "arm64" {
+		// https://student.cs.uwaterloo.ca/~cs452/docs/rpi4b/aapcs64.pdf
```

Comment:
Isn't https://github.com/ARM-software/abi-aa a more relaiable and stable URL for the PDFs?